spec/webdrivers/chromedriver_spec.rb

```diff
@@ -66,7 +66,7 @@
         allow(Net::HTTP).to receive(:get_response).and_raise(SocketError)
         allow(chromedriver).to receive(:exists?).and_return(false)
 
-        msg = %r{Can not reach https://chromedriver.storage.googleapis.com}
+        msg = %r{Can not reach https://googlechromelabs.github.io/chrome-for-testing/latest-patch-versions-per-build.json}
```

Comment:
How about considering the both cases for v115+ and v114 or earlier? The Gem is expected to print "Can not reach ...", and the target URL is not important here.

```suggestion
        msg = %r{Can not reach (?:https://googlechromelabs.github.io|https://chromedriver.stor
age.googleapis.com)}
```